src/js/utils/LocalStorageUtil.ts

```diff
@@ -0,0 +1,17 @@
+const LocalStorageUtil = {
+  set(_key: string, _value: unknown) {
+    return (global as any).localStorage.setItem.apply(
+      (global as any).localStorage,
+      arguments
```

Comment:
why not use `_key` and `_value` here?

i see no point in using `apply`. i know it was this way before, but maybe we can change that due to the transition to typescript?

Reply:
I was wondering about that as well. But I was more focused on the conversion and not refactoring. I am happy to refactor it.

Reply:
done